### PR TITLE
VZ-11582: Add more detail to OAM to Kubernetes guide

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 1]
```diff
@@ -22,6 +22,29 @@ $ vz export oam --name hello-helidon --namespace hello-helidon > myapp.yaml
 </div>
 {{< /clipboard >}}
 
+The output of the command `vz export oam` can be used to deploy the application on another cluster.
```
Comment (rosemarymarano) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:25]: -> You can use the output of the command `vz export oam` to deploy the application on another cluster.

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 2]
```diff
@@ -22,6 +22,29 @@ $ vz export oam --name hello-helidon --namespace hello-helidon > myapp.yaml
...
+<div class="highlight">
+
+```
+$ kucbectl create namespace hello-helidon
+$ kubectl apply -f myapp.yaml
+```
+</div>
+{{< /clipboard >}}
+
+The output of the command `vz export oam` can be edited before deploying the application.  The extent to which the exported YAML may be edited will vary based on local requirements. Here are some examples of changes that may be made to the exported YAML.
```
Comment (rosemarymarano) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:37]: -> In addition, you can edit the output of the command `vz export oam` before deploying the application.
->  made to the exported YAML: (colon)

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 3]
```diff
@@ -22,6 +22,29 @@ $ vz export oam --name hello-helidon --namespace hello-helidon > myapp.yaml
...
+* Authorization policies
+* Values for secrets
+* Mount volume definitions
+* Replica counts
+* Prometheus logging rules
+
 ## Overview of OAM to Kubernetes Mappings
 
-The following documents show the Kubernetes objects generated for each OAM object.
+The following documents show the Kubernetes objects generated for each OAM object.  It is recommended to start with the output of command `vz export oam` and edit the YAML as needed.  However, this section is provided to give some insight into how each OAM object is converted into one or more Kubernetes objects.
```
Comment (rosemarymarano) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:50]: -> However, we recommend that you start with the output of the `vz export oam` command and edit the YAML as needed. 
-> (Remove) However, this section is provided to give some insight into how each OAM object is converted into one or more Kubernetes objects. (This sentence is redundant with the first sentence and confusing; I'd suggest removing it. If you want to explain to me why it's needed (or what you mean by "this section"), please ping me and we'll discuss. )

**Review by rosemarymarano — COMMENTED** [review]
comments/suggestions

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 1 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md]: fixed

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 2 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md]: fixed

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 3 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md]: I fixed it by using the paragraph you suggested.  Thank you!

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 4]
```diff
@@ -47,4 +47,4 @@ The output of the command `vz export oam` can be edited before deploying the app
 
 ## Overview of OAM to Kubernetes Mappings
 
-The following documents show the Kubernetes objects generated for each OAM object.  It is recommended to start with the output of command `vz export oam` and edit the YAML as needed.  However, this section is provided to give some insight into how each OAM object is converted into one or more Kubernetes objects.
+The following documents show the Kubernetes objects generated for each OAM object. It is provided to give you some insight into how each OAM object is converted into one or more Kubernetes objects. However, Oracle recommends that you first start with the output of the `vz export oam` command and then edit the YAML as needed.
```
Comment (rosemarymarano) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:50]: Sorry, one grammar mistake that was mine:
-> They are provided to give you some insight

**Review by rosemarymarano — APPROVED** [review]
One more fix because of me, otherwise fine.

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 4 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md]: Thank you, I pushed a fix

**Pull request merged**